<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnRFID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>RFID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddRecord">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>380</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Record</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>0</y>
      <width>801</width>
      <height>371</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Burnt Calorie</string>
     </attribute>
     <widget class="QLabel" name="lCal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>771</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lDRank1">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>310</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Today rank:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lGRank1">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>310</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Global rank:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Point</string>
     </attribute>
     <widget class="QLabel" name="lPoint">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>771</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lGRank2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>310</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Global rank:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lDRank2">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>310</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Today rank:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Burnt Calorie Improvment</string>
     </attribute>
     <widget class="QLabel" name="lProgCal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>771</width>
        <height>311</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Point Improvment</string>
     </attribute>
     <widget class="QLabel" name="lProgPoint">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>771</width>
        <height>301</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Global Burnt Calorie</string>
     </attribute>
     <widget class="QLabel" name="lGRank3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>310</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Global rank:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lDRank3">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>310</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Today rank:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lGloabalCal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>771</width>
        <height>301</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
